packages/toast/src/Toast.story.tsx

```diff
@@ -7,27 +7,33 @@ import Button from '@leafygreen-ui/button';
 import { css } from '@leafygreen-ui/emotion';
 import LeafyGreenProvider from '@leafygreen-ui/leafygreen-provider';
 import { DarkModeProps, StoryMeta } from '@leafygreen-ui/lib';
+import { ToastProvider, useToast, Variant } from '@leafygreen-ui/toast';
 import { InlineCode, Label } from '@leafygreen-ui/typography';
 
 import { variantIcons } from './InternalToast/VariantIcon';
 import { makeToast, makeToastStack } from './ToastContext/utils/makeToast';
 import { InternalToast, InternalToastProps } from './InternalToast';
-import { ToastProvider, useToast, Variant } from '.';
 
 export default StoryMeta<typeof InternalToast>({
   title: 'Components/Toast',
   component: InternalToast,
   decorators: [
     (Story, meta) => (
       <LeafyGreenProvider darkMode={!!meta.args.darkMode}>
-        <ToastProvider initialValue={meta.args.initialValue}>
+        <ToastProvider
+          initialValue={meta.args.initialValue}
+          portalClassName={css`
+            position: relative;
+            z-index: 1;
+          `}
+        >
```

Comment:
If this is just necessary for the LiveExample, it'd be helpful to have some comments on why this is necessary